jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.c

```diff
@@ -24,6 +24,8 @@
 #include "fdlibm-math.h"
 #include "lit-char-helpers.h"
 
+#include <time.h>
+
 #ifndef CONFIG_ECMA_COMPACT_PROFILE_DISABLE_DATE_BUILTIN
```

Comment:
I think the <time.h> include should be inside the ifndef. It is not needed otherwise.